### ISSUE TITLE
Genesis block overwritting

### ISSUE THREAD

**Issue by l4l** [hyperledger/iroha#1231]
For now if genesis block provided for `irohad` it will be silently overwritten. It is not really good behavior and user should be asked for confirmation.

Moreover the flag for accept all confirmations should be added with that fix for related test

**Comment by ZeekHuge** [comment]
Hi, I would like to work to resolve this issue.

I have done some playing using docker containers and network, using the `develop` branch.
Also, after about 4 successfully commited transactions, I compared the provided genesis block (in my case the [example genesis block](https://github.com/hyperledger/iroha/blob/master/example/genesis.block)) with the one stored at the `/tmp/block_store/0000000000000001` and it looks to be exactly same (and not being overwritten)

Can you help me to understand what the problem is ?

**Comment by grimadas** [comment]
Hi @ZeekHuge. We would love to see your pr. This issue can happen if you already have some blocks in your blockstore and you try to  run `irohad` with a new genesis block that is different than `/tmp/block_store/0000000000000001`. This will overwrite it. 
One possible solution is to give a warning that genesis block is being overwritten and ask user for confirmation

**Comment by ZeekHuge** [comment]
Hi @grimadas  ! 
I tried to replicate this behavious, but what I found was that _everything_ is removed when `irohad` is restarted. The block_store is just emptied. And this looks like the right behaviour - according to [this line here](https://github.com/hyperledger/iroha/blob/master/irohad/main/irohad.cpp#L146). 

Is it that we want a warning and a confirmation before everything is deleted or something like that ? 
Note that I am using `develop` branch, can that be a possible reason why I am getting a different behaviour ?
